backend/routers/resources.js

```diff
@@ -1,39 +1,22 @@
 const { Router } = require('express');
+const { ResourceModel } = require('../models');
 
 const router = Router();
 
-const mockedResources = {
-  resources: [
-    {
-      type: 'food',
-      amount: 1,
-      generation: 60,
-      updatedAt: 1353647,
-    },
-    {
-      type: 'gold',
-      amount: 1,
-      generation: 60,
-      updatedAt: 1235346,
-    },
-  ],
-};
-
-function getResourceAmount(initialAmount, generation, updatedAt) {
-  const date = new Date();
-  const minutesPassed = (date.getTime() - updatedAt) / 60000;
-  return Math.round(initialAmount + generation * minutesPassed);
-}
-
-function getResources(req, res) {
-  const calculatedResources = {
-    resources: mockedResources.resources.map((resource) => ({
-      type: resource.type,
-      amount: getResourceAmount(resource.amount, resource.generation, resource.updatedAt),
-      generation: resource.generation,
-    })),
-  };
-  return res.status(200).send(calculatedResources);
+async function getResources(req, res, next) {
+  try {
+    const result = await ResourceModel.find({
+      owner: 1,
+    });
+    const resultFiltered = result.map((e) => ({
```

Comment:
consider using destructing like this.
```suggestion
    const resultFiltered = result.map(({ type, generation, amount}) => ({ type, generation, amount });
```